examples/slimevolley/train_rainbow.py

```diff
@@ -176,7 +176,7 @@ def phi(x):
     # Turn off explorer
     explorer = explorers.Greedy()
 
-    # Use the same hyper parameters as https://arxiv.org/abs/1707.06887
+    # Use the same eps as https://arxiv.org/abs/1710.02298
```

Comment:
Shouldn't we remove this comment? It doesn't use the same learning rate as `train_rainbow.py`:

![image](https://user-images.githubusercontent.com/10005453/89368732-34e71300-d717-11ea-8905-08af858738d7.png)

Reply:
I kept it since we're still taking the eps value from there, I guess that was the original idea

Reply:
Hmm. Sure.